server/routes/place/html.py

```diff
@@ -30,7 +30,7 @@
 
 
 @bp.route('', strict_slashes=False)
-@bp.route('/<path:place_dcid>', strict_slashes=False)
+@bp.route('/<path:place_dcid>/', strict_slashes=False)
```

Comment:
curious why we need the extra slash

Reply:
The latest Flask can not handle the trailing slash without specifying it explicitly.